### DISCUSSION TITLE
Can't register on Azure with the Student Pack offer

### DISCUSSION THREAD

**Discussion by Mudb0y** [community/community#33074] [GitHub Education]
Hello,
I just applied for my Student Pack yesterday and got access. I'm a verified student now, but when I try to register on the Azure offer the website says I'm not verified.
Tried contacting Azure support but got no valid help from there.
Do I have to wait a few days / weeks for Microsoft to update their stuff or something?
Thanks for any help.

**Comment by samuel-ma** [comment]
@Mudb0y 
Am also using the student pack and registered on Azure
I did so two days after getting my student pack though
Try again in a few days, it should work just fine...
